packages/pyolite-kernel/src/worker.ts

```diff
@@ -24,6 +24,7 @@ async function loadPyodideAndPackages() {
     import micropip
     await micropip.install([
       'traitlets',
+      'importhook',
```

Comment:
We can track how to best handle these extra packages in https://github.com/jupyterlite/jupyterlite/issues/45#issuecomment-878103407, so they can be configurable and not prevent offline use.